webpack.config.js

```diff
@@ -132,22 +108,50 @@ if (ENV === 'development') {
   console.log('webpack dev');
 
   module.exports = merge(commonSettings, {
+    name: 'client',
     mode: 'development',
     devtool: 'inline-source-map',
-    entry: {
-      app: [
-        'webpack-dev-server/client?http://localhost:3000',
-        'webpack/hot/only-dev-server',
-      ],
-    },
-    output: {
-      publicPath: 'http://localhost:3000/',
-    },
-    plugins: [new webpack.HotModuleReplacementPlugin()],
+    stats: 'errors-only',
+    // Keep for Past Context
```

Comment:
Can you elaborate a little bit on what "past context" should be, especially for a future audience that is not you?

Reply:
Sure! I'll add more context. Though I am planning to refactor this page and clean up the env configuration.

Reply:
ah gotcha.  I'd still add enough context that another person (or you, eight months from now), would know what this was here for.  Much better to have it and not need it than to be cursing your past self.